traveler-swift-core/TravelerKit/Models/Product.swift

```diff
@@ -26,20 +26,40 @@ public enum ProductType: String, Decodable {
 
 struct AnyProduct: Decodable {
     let id: String
-    //let price: Price
+    let price: Price
     let title: String
     let productType: ProductType
 
     // Properties for BookableProduct
     let passes: [Pass]?
+    let eventDate:Date
```

Comment:
`let eventDate: Date`....Swift convention leave one whitespace between `:` and type